workers/loc.api/queue/helpers.js

```diff
@@ -475,8 +478,8 @@ const _getCompleteFileName = (
     : formattedDateNow
   const _ext = ext ? `.${ext}` : ''
   const _userInfo = userInfo ? `${userInfo}_` : ''
-  const fileName = queueName === 'getWallets'
-    ? `${_userInfo}${baseName}_MOMENT_${formattedDateNow}${_ext}`
+  const fileName = queueName === 'getWallets' || isMultiExport
+    ? `${_userInfo}${baseName}_MOMENT_${isMultiExport ? timestamp : formattedDateNow}${_ext}`
```

Comment:
As to make it simpler can both on getWallets and multiExport use `formattedDateNow`